pages/learn.vue

```diff
@@ -107,20 +123,23 @@ import { Component } from 'vue-property-decorator'
 import QiskitPage from '~/components/logic/QiskitPage.vue'
 import LearningResourceCard from '~/components/learn/LearningResourceCard.vue'
 import TheCarefulExplanationForBeginner from '~/components/learn/TheCarefulExplanationForBeginner.vue'
-import TheCarefulExplanationForAdvanced from '~/components/learn/TheCarefulExplanationForAdvanced.vue'
+import TheCarefulExplanationForExperts from '~/components/learn/TheCarefulExplanationForExperts.vue'
+import AppLink from '~/components/ui/AppLink.vue'
 import {
   TimeScale,
   LEARN_LEVELS,
   TIME_SCALES,
   LEARN_LEVEL_OPTIONS,
-  TIME_SCALE_OPTIONS
+  TIME_SCALE_OPTIONS,
+  LearnLevel
 } from '~/store/modules/learning-resources.ts'
 
 @Component({
   components: {
     LearningResourceCard,
-    TheCarefulExplanationForAdvanced,
-    TheCarefulExplanationForBeginner
+    TheCarefulExplanationForExperts,
```

Comment:
We are using "Experts" and "Beginner", would you mind to add an "s" to "beginner" to follow the same structure?